include/swift/AST/AccessScope.h

```diff
@@ -0,0 +1,89 @@
+//===--- AccessScope.h - Swift Access Scope ---------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the AccessScope class.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_ACCESSSCOPE_H
+#define SWIFT_ACCESSSCOPE_H
+
+#include "llvm/ADT/PointerIntPair.h"
+#include "swift/AST/AttrKind.h"
+#include "swift/AST/DeclContext.h"
+
+namespace swift {
+
+/// AccessScope - The wrapper around the outermost DeclContext from which
```

Comment:
Nitpick: New Doxygen comments should not have a leading "Name - " and should not use `\brief` unless matching other comments in the same file. Neither of these are necessary in Doxygen these days.

(At some point I'll go on a rampage and clean up all the bad habits we inherited from old LLVM code.)